src/app/pages/things/opcua/opcua.component.html

```diff
@@ -30,47 +21,55 @@
       </div>
     </div>
   </nb-card-header>
+</nb-card>
+
+<nb-card size="medium">
+  <nb-card-header>
+    <nb-list-item>
```

Comment:
Don't know if `nb-list-item` is implicitely a row or we need to say it explicitely.

Reply:
it's implicit